rllib/tests/agents/parameters.py

```diff
@@ -212,117 +212,119 @@ def astuple(self):
+        #     },
+        #     n_iter=200,
+        #     threshold=150.0,
+        # ),
+        # TestAgentParams.for_cart_pole(
+        #     algorithm=DiscreteActionSpaceAlgorithm.SAC,
+        #     config_updates={
+        #         "num_workers": 4,
+        #         "twin_q": True,
+        #         "soft_horizon": True,
+        #         "clip_actions": False,
+        #         "normalize_actions": True,
+        #         "learning_starts": 0,
+        #         "prioritized_replay": True,
+        #         "Q_model": {"fcnet_hiddens": [64, 64]},
+        #         "policy_model": {"fcnet_hiddens": [64, 64],},
+        #     },
+        #     n_iter=200,
+        #     threshold=100.0,
+        # ),
+        # TestAgentParams.for_cart_pole(
+        #     algorithm=ContinuousActionSpaceAlgorithm.APEX_SAC,
+        #     config_updates={
+        #         "seed": 42,
+        #         "num_workers": 8,
+        #     },
+        #     n_iter=100,
+        #     threshold=175.0,
+        # ),
         TestAgentParams.for_pendulum(
             algorithm=ContinuousActionSpaceAlgorithm.APEX_SAC,
             config_updates={
                 "num_workers": 8,
+                # "num_envs_per_worker": 2,
                 "exploration_config": {"type": "StochasticSampling"},
-                "no_done_at_end": True,
+                "prioritized_replay": False,
             },
             # TODO: Delete next line before landing PR
+            frameworks=[Framework.TensorFlow],
             n_iter=200,
             threshold=-350.,
         ),
-        TestAgentParams.for_pendulum(
-            algorithm=DiscreteActionSpaceAlgorithm.SAC,
-            config_updates={
-                "horizon": 200,
-                "soft_horizon": True,
-                "Q_model": {"fcnet_activation": "relu", "fcnet_hiddens": [256, 256]},
-                "policy_model": {
-                    "fcnet_activation": "relu",
-                    "fcnet_hiddens": [256, 256],
-                },
-                "tau": 0.005,
-                "target_entropy": "auto",
-                "no_done_at_end": True,
-                "n_step": 1,
-                "rollout_fragment_length": 1,
-                "prioritized_replay": True,
-                "train_batch_size": 256,
-                "target_network_update_freq": 1,
-                "timesteps_per_iteration": 1000,
-                "learning_starts": 256,
-                "optimization": {
-                    "actor_learning_rate": 0.0003,
-                    "critic_learning_rate": 0.0003,
-                    "entropy_learning_rate": 0.0003,
-                },
-                "num_workers": 4,
-                "num_gpus": 0,
-                "clip_actions": False,
-                "normalize_actions": True,
-                "metrics_smoothing_episodes": 5,
-            },
-            n_iter=200,
-            threshold=-750.0,
-        ),
+        # TestAgentParams.for_pendulum(
```

Comment:
Question: What's this test for? do we want to comment it out?

Reply:
Good point. I need to uncomment them and add the test with prioritized replay enabled too.

Reply:
We shouldn't comment this out. I fixed this in the update. Thanks for catching it.